tools/docs/config.ts

```diff
@@ -233,10 +237,65 @@ export async function generateConfig(dist: string, bot = false): Promise<void> {
         `\n${option.description}\n\n` +
         genTable(Object.entries(el), option.type, option.default);
 
+      if (el.advancedUse) {
+        if (option.name === 'excludeCommitPaths') {
+          configOptionsRaw[headerIndex] += generateAdvancedUse();
+        } else {
+          configOptionsRaw[footerIndex] += generateAdvancedUse();
+        }
+      }
+
+      if (el.followTagAdvancedFunctionality) {
+        configOptionsRaw[headerIndex] +=
+          generateFollowTagAdvancedFunctionality();
+      }
+
+      if (el.groupAdvancedFunctionality) {
+        configOptionsRaw[headerIndex] += generateGroupAdvancedFunctionality();
+      }
+
+      if (el.advancedConfig) {
+        configOptionsRaw[footerIndex] += generateAdvancedConfig();
+      }
+
+      if (el.directEditOfCommitMessageDeprecation) {
+        configOptionsRaw[footerIndex] += generateCommitMsg();
+      }
+
       if (el.experimental) {
         configOptionsRaw[footerIndex] += genExperimentalMsg(el);
       }
     });
 
   await updateFile(`${dist}/${configFile}`, configOptionsRaw.join('\n'));
 }
+
+function generateAdvancedUse(): string {
+  return (
+    '\n<!-- prettier-ignore -->\n!!! warning\n' +
+    '    For advanced use only! Use at your own risk!\n'
+  );
+}
+
+function generateFollowTagAdvancedFunctionality(): string {
+  return "\n<!-- prettier-ignore -->\n!!! warning\n    Advanced functionality.\n    Only use this if you're sure you know what you're doing.\n";
```

Comment:
```suggestion
  return "\n<!-- prettier-ignore -->\n!!! warning\n    Advanced functionality.\n    Only use this if you know what you're doing.\n";
```

Is there a reason that `generateFollowTagAdvancedFunctionality` and `generateGroupAdvancedFunctionality` have different string messages? I'm not quite sure what the difference is.

If we need the different messages, then ignore the suggestions that change these strings. 😉

Reply:
I think `generateFollowTagAdvancedFunctionality` and `generateGroupAdvancedFunctionality`  should return the same string.